src/css.js

```diff
@@ -113,6 +113,13 @@ function getWidthOrHeight( elem, name, extra ) {
 		styles = getStyles( elem ),
 		isBorderBox = jQuery.css( elem, "boxSizing", false, styles ) === "border-box";
 
+	// Support: IE11+
+	// Fix for edge case in IE 11. See https://github.com/jquery/jquery/issues/1764
+	if ( !support.reliableDimensionsInFullscreenMode && window.top !== window.self &&
+		document.msFullscreenElement ) {
+		val = Math.round( elem.getBoundingClientRect()[ name ] * 100 );
```

Comment:
Yes, that's intended, it has worked this way in all browsers since almost forever. ;) We don't support fractional width/height anywhere for now, #1724 is supposed to change that but until that happens we need to be consistent.